### PR TITLE
Add subsumption and multiple authors to notifications

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

File: packrat/scripts/notices.js [hunk 1]
```diff
@@ -29,16 +29,35 @@ function renderNotices($container, isAdmin) {
...
+  }
+}
+
 function getRenderedNotices(notices, $notifyPane, callback) {
   var renderedNotices = [];
   var done = 0;
   $.each(notices, function (i, notice) { 
     if (~NOTICE_TYPES.indexOf(notice.category)) {
+      var authors = notice.details.authors || [notice.details.author]
+      var authorName = formatAuthorNames(authors);
```
Comment (2is10) [inline packrat/scripts/notices.js]: this line appears to be dead code

File: packrat/scripts/notices.js [hunk 2]
```diff
@@ -29,16 +29,35 @@ function renderNotices($container, isAdmin) {
   });
 }
 
+function formatAuthorNames(authors) {
+  var names = authors.length > 1 ?
+    authors.map(function (a) { return a.firstName; }) :
+    [authors[0].firstName + " " + authors[0].lastName];
+  names = names.map(function (n) {
+    return $('<span class="author-name">').text(n).wrapAll('<span>').parent().html();
```
Comment (2is10) [inline packrat/scripts/notices.js:37]: what's the extra `<span>` wrapper for?

**Comment by 2is10** [comment]
Nice. Looks good. Feel free to merge.

File: packrat/scripts/notices.js (hunk 1 above)
Comment (gmethvin) [inline packrat/scripts/notices.js]: thanks

File: packrat/scripts/notices.js (hunk 2 above)
Comment (gmethvin) [inline packrat/scripts/notices.js:37]: It's an easy way to escape the content as html.

Comment (2is10) [inline packrat/scripts/notices.js:37]: ah, yes, I was forgetting that `.html()` returns innerHTML. you could also shorten this to: `$(...).text(n)[0].outerHTML`.
